tools/istio-iptables/pkg/cmd/run.go

```diff
@@ -145,12 +145,20 @@ func (iptConfigurator *IptablesConfigurator) handleInboundPortsInclude() {
 				//TODO: (abhide): Move this out of this method
 				iptConfigurator.ext.RunOrFail(constants.IP, "route", "show", "table", "all")
 			}
+
+			ifaceName := getNIC()
```

Comment:
how do we know the first interface is the right one?

Reply:
In most case, only one ip/nic. For multi addresses scenario, we have cni-genie env. And it turns out that the first nic is the right one.
https://asciinema.org/a/118191